builtin/providers/aws/resource_aws_rds_cluster_instance.go

```diff
@@ -58,13 +58,31 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 				Default:  false,
-				ForceNew: true,
 			},
 
 			"instance_class": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
-				ForceNew: true,
+			},
+
+			"db_parameter_group_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"status": &schema.Schema{
```

Comment:
This attribute doesn't seem to be set anywhere, what is it meant to record? The `DBInstanceStatus`, or `status` from `StatusInfos.DBInstanceStatusInfo.N` ?

- http://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBInstance.html